pyvecorg/avatars.py

```diff
@@ -0,0 +1,43 @@
+import os
+from urllib.parse import quote
+import hashlib
+
+from PIL import Image, ImageFilter
+import requests
+
+
+def get_avatar_url(member):
+    no_avatar = 'https://www.gravatar.com/avatar/0000?d=mm&f=y'
+    try:
+        key = member['avatar']
+        value = member[key]
+    except KeyError:
+        return no_avatar
+
+    if key == 'email':
+        email_hash = hashlib.md5(value.lower().strip().encode()).hexdigest()
+        return f'https://www.gravatar.com/avatar/{email_hash}?size=400&d=404'
```

Comment:
why getting size 400 if you resize it to 100?

Reply:
good catch